### DISCUSSION TITLE
Need help to run tests on windows

### DISCUSSION THREAD

**Discussion by MobiliteDev** [invertase/dart_custom_lint#292] [Q&A] [answered]
Hello,

I'm encountering an issue running tests on the main branch using the Dart main channel on Windows, where most tests fail with the following error:

![image](https://github.com/user-attachments/assets/526a12c5-dfb4-480b-8292-639c4e8d65dd)

It seems the temporary folder for the tests is located on my `C:\` drive. When parsing the file location to create a URI, it's resolved with a `c` scheme (from "C:\") instead of the expected `file` scheme.

The problem appears to happen when `projectPackageConfig` is created via `parsePackageConfig` (in `workspace.dart`, line 920):

```dart
final projectPackageConfig = await parsePackageConfig(projectDirectory);
```

Within this code, `Uri.parse(rootUri)` interprets the Windows path (`C:\Users\MyUser\AppData\Local\Temp\custom_lint_test89169a03\plugin1`) as a `c` scheme, rather than `file`.

I realize this may not be directly related to this package, as it involves core Dart URI handling. However, I'd like to know if others have experienced this issue or if there are any workarounds to make this work on Windows. How do you run these tests on Windows without encountering this issue?

Thanks

**Comment by rrousselGit** [comment, marked as answer]
I do not code on a windows device, so I never ran tests on windows. It could be that they don't work there.
